src/dlls/mscorpe/ceefilegenwriter.cpp

```diff
@@ -420,13 +427,13 @@ HRESULT CeeFileGenWriter::generateImage(void **ppImage)
 
     if (! outputFileName && ppImage == NULL) {
         if (m_comImageFlags & COMIMAGE_FLAGS_IL_LIBRARY)
-            outputFileName = L"output.ill";
+            outputFileName = outputFileNameIll;
```

Comment:
Btw, the problem is that the write method parameter uses "const LPWSTR" type, which is not what it really wanted. const LPWSTR makes the string pointer constant, not the string characters themselves. LPCWSTR would do the trick.